web_src/less/_markup.less

```diff
@@ -1,4 +1,4 @@
-.markdown:not(code) {
+.markup:not(code) {
```

Comment:
Maybe drop this `:not(code)` and see what breaks, if anything at all? Might need to check git history why it was added in first place.

Reply:
```
rg markdown.+code templates
rg code.+markdown templates
```
No relevant matches.

Reply:
Git history points to it being related to webhook:

https://github.com/go-gitea/gitea/commit/c2ca103d30b0f47b5ad0fa544ab5a07263894ea3

Reply:
Webhook delivery list looks fine with the removal, let's do this.

Reply:
Pushed this change.